report/metadata_template.go

```diff
@@ -124,11 +124,14 @@ func (e MetadataTemplates) Copy() MetadataTemplates {
 // Merge merges two sets of MetadataTemplates so far just ignores based
 // on duplicate id key
 func (e MetadataTemplates) Merge(other MetadataTemplates) MetadataTemplates {
-	result := e.Copy()
+	if e == nil && other == nil {
+		return nil
+	}
+	result := make(MetadataTemplates, len(e))
```

Comment:
Then, this initialization and loop below are equivalent to `result := e.Copy` after adding `make(MetadataTemplates, len(e))` to `Copy()`.